install.sh

```diff
@@ -88,6 +88,11 @@ function ask_confirmation {
     done
 }
 
+# Check if we have doas, if we do, use it instead of sudo.
+if [[ -z "$(command -v doas)" ]]; then
+	SUDO=doas
+fi
+
 # Usually we use sudo, but if prefix is somewhere in ~/, we don't need sudo
 SUDO=sudo
```

Comment:
The former won't do anything, because SUDO is set to sudo here anyway.

Reply:
I think the changes I made just now should be good.